storage/container_storage_interface/persistent-storage-csi-azure.adoc

```diff
@@ -0,0 +1,39 @@
+[id="persistent-storage-csi-azure-disk"]
+= Azure Disk CSI Driver Operator
+include::modules/common-attributes.adoc[]
+:context: persistent-storage-csi-azure
+
+toc::[]
+
+== Overview
+
+{product-title} is capable of provisioning persistent volumes (PVs) using the Container Storage Interface (CSI) driver for Microsoft Azure Disk Storage.
+
+:FeatureName: Azure Disk Driver Operator
+include::modules/technology-preview.adoc[leveloffset=+1]
+
+Familiarity with xref:../../storage/understanding-persistent-storage.adoc#understanding-persistent-storage[persistent storage] and xref:../../storage/container_storage_interface/persistent-storage-csi.adoc#persistent-storage-csi[configuring CSI volumes] is recommended when working with a CSI Operator and driver.
+
+To create CSI-provisioned PVs that mount to Azure Disk storage assets, {product-title}, after this feature is enabled, installs the Azure Disk CSI Driver Operator and the Azure Disk CSI driver by default in the `openshift-cluster-csi-drivers` namespace.
+
+* The _Azure Disk CSI Driver Operator_ , after being enabled, provides a storage class (`managed-csi`) that you can use to create persistent volume claims (PVCs). The Azure Disk CSI Driver Operator supports dynamic volume provisioning by allowing storage volumes to be created on-demand, eliminating the need for cluster administrators to pre-provision storage.
```

Comment:
Same question, could you leave out "after being enabled"? This is just describing the feature anyway. 🤷

Reply:
Nope, sadly. As I mentioned above QE/Engr really want to stress the enabling part because this is new for storage. If you don't enable TP, you don't get the feature. They are very worried users will trip over this.